Update 101-after-js.md added introduction and links (#1210)
kamranahmedse/developer-roadmap: content/roadmaps/100-frontend/content/118-server-side-rendering/100-react-js/101-after-js.md

> # After js
> 
> After.js is an opensource javascript framework for developing SSR (Server Side Rendering) based applications.
> Similar to Next.js framework for server-rendered React app but uses React Router instead of a folder structure based router like Next.js
> 
> <ResourceGroupTitle>Free Resources</ResourceGroupTitle>
> <BadgeLink colorScheme='blue' badgeText='GitHub' href='https://github.com/jaredpalmer/after.js'>Official GitHub repository</BadgeLink>
> <BadgeLink colorScheme='blue' badgeText='Docs' href='https://www.npmjs.com/package/@jaredpalmer/after'>Documetation</BadgeLink>

Reviewer: ```suggestion
# After.js

After.js is an open-source JavaScript framework for developing SSR (Server Side Rendering) based applications. It is similar to the Next.js framework for server-rendered React apps but uses React Router instead of a folder structure based router like Next.js

<ResourceGroupTitle>Free Resources</ResourceGroupTitle>
<BadgeLink colorScheme='blue' badgeText='GitHub' href='https://github.com/jaredpalmer/after.js'>Official GitHub repository</BadgeLink>
<BadgeLink colorScheme='blue' badgeText='Docs' href='https://www.npmjs.com/package/@jaredpalmer/after'>Documetation</BadgeLink>
```
